hBank/src/main/java/animalPark/Feline.java

```diff
@@ -0,0 +1,7 @@
+package animalPark;
+
+public class Feline extends Animal{
```

Comment:
How can we make sure that it's not possible to instantiate objects of this class?

Reply:
Done - abstract class